CMakeLists.txt

```diff
@@ -1,22 +1,16 @@
-cmake_minimum_required(VERSION 3.8)
-
-include(GNUInstallDirs)
+cmake_minimum_required(VERSION 3.13)
```

Comment:
I think it would be good to move to "modern CMake". There doesn't seem to be an exact version when it starts, but it is around 3.11+, 3.12+:

  https://cliutils.gitlab.io/modern-cmake/

It looks like 3.13 was chosen because it is the first to support `add_link_options`.

Reply:
Yah, add_link_options is definitely something we'd prefer to have. I'm fine with this, but we'll need to confirm that it's not breaking anything for others.